libs/langchain/langchain/utilities/promptguard.py

```diff
@@ -0,0 +1,80 @@
+import json
+from typing import Dict, Union
+
+
+def sanitize(
```

Comment:
Where are these functions actually called?

Reply:
They are called in 
1. the notebook demo
2. the integration test.